docs/UserGuide.md

```diff
@@ -474,9 +474,15 @@ use the `unremark` command to delete it._
 Directly use the remark command and it overwrites the old remark!
 </div>
 
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes:**<br>
+* You can unremark a company that already have no remarks. 
+LinkMeIn does not check if a remark exists i.e the same result message will be displayed regardless of whether a remark exist for a company.
```

Comment:
You can unremark a company that already has no remarks. LinkMeIn does not check for the existence of a remark. The same message will be displayed regardless of whether a remark exists for a company.


Will this phrasing be better

Reply:
Maybe specify that the "same message" is the output message